tex-exchange-contract/src/main/kotlin/com/ccc/state/Stock.kt

```diff
@@ -0,0 +1,47 @@
+package com.ccc.state
+
+import com.ccc.contract.StockContract
+import net.corda.core.contracts.BelongsToContract
+import net.corda.core.contracts.LinearState
+import net.corda.core.contracts.UniqueIdentifier
+import net.corda.core.identity.AbstractParty
+import net.corda.core.identity.Party
+
+/**
+ * A [LinearState] describing a Stock.
+ * @param description Textual description of the  item.
+ * @param owner The Party who owns the  item.
+ * @param listed Whether or not the  item is listed in an active sale.
+ * @param linearId Unique identifier of a StockState object.
+ */
+@BelongsToContract(StockContract::class)
+data class Stock
+    (
+    override val linearId: UniqueIdentifier = UniqueIdentifier(),
+    val description: String,
+    val owner: Party,
+    val listed: Boolean = false
+) : LinearState {
```

Comment:
This might be better modelled as a `FungibleAsset` e.g. in its current form we have no concept of `quantity`

Reply:
Yes, I will try to make this Fungible Asset.  Also, I will add quantity

Reply:
On second thought `FungibleToken` from Tokens SDK is probably the right thing to use here.

Reply:
sure